src/App.js

```diff
@@ -1,14 +1,19 @@
-import logo from './logo.svg';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
+import Navbar from './Components/Navbar';
+import CountriesCountainer from './Pages/CountriesContainer';
+import Region from './Pages/RegionContainer';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <header>
+      <Navbar />
+    </header>
+    <Routes>
+      <Route exact path="/" element={<CountriesCountainer />} />
+      <Route path="/:country" element={<Region />} />
+    </Routes>
+  </Router>
+);
```

Comment:
- [ ] Please, consider that when the app opens the first time it is not opening the right page. As you can see on the gif, I have to click back to see the list of countries. On the first opened page the "loading" status is available. Let us fix this issue to cover the project requirements 👍 
![2022-05-27 10 25 49](https://user-images.githubusercontent.com/49064106/170643459-17d8c9da-177f-4998-9352-f29b6a72e8bf.gif)

Reply:
HI @elmar8287 
Thank you, for your feedback, I have implemented requested changes.